backend/mora/common.py

```diff
@@ -56,6 +56,18 @@ class RequestType(enum.Enum):
     EDIT = 1
 
 
+# Handlers is populated by each individual active RequestHandler
```

Comment:
Totally picking a nit, but “Handlers is” sounds really weird. “The "Handlers" mapping is”?